### PR TITLE
[Documentation] update issue template

### PR DESCRIPTION
**Pull request by iaulakh**
## Purpose

* update issue template

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 
-**In what environment did you see the issue?**
+**AzureCommunicationUILibrary (please complete the following information):**
+ - Version [e.g. 22]
 
-- OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
-- Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
-- Version [e.g. 1.8.0]
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
```
Comment (AmyL219) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:30]: Probably need to update here to Android device

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
 
-**What are the steps to reproduce the issue?**
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (AmyL219) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:18]: I think it's a good idea to add how often could this bug be reproduced. Something like
```
....
4. See error
This bug is [always/occasional/seldom] reproducible. 
```

**Review by AmyL219 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 3]
```diff
@@ -1,23 +1,34 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
 
-**If applicable, provide screenshots:**
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
 
-**In what environment did you see the issue?**
+**AzureCommunicationUILibrary (please complete the following information):**
+ - Version [e.g. 22]
```
Comment (AmyL219) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:27]: My bad. We should update to 
```
- Version [e.g. 1.0.0-beta.1]
```

**Review by ShaunaSong — APPROVED** [review]

**Review by Mrayyan — APPROVED** [review]

**Review by ahammer — APPROVED** [review]
Approving since good enough and Inderpal isn't around to update.

@AmyL219 if you can approve if your otherwise happy we can further amend later.

**Comment by ahammer** [comment]
Merged because 3 approvals now and Inderpal is on vacation can't address comments. Future comments can be done in a fresh PR.

**Pull request merged**